common/assets/scss/overrides/_positioning.scss

```diff
@@ -5,3 +5,15 @@
 .app-\!-float-left {
   float: left !important;
 }
+
+.app-\!-position-relative {
+  position: relative !important;
+}
+
+@media (min-width: 40.0625em) {
+  .app-\!-position-top-right {
+    position: absolute;
+    top: 0;
+    right: 0;
+  }
+}
```

Comment:
Seems to be missing a new line again?

Reply:
Think there's a few files, worth checking and maybe setting up something like .editorconfig to automatically add this.